spec/features/edit_course_details_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+feature "edit course details" do
```

Comment:
I've nested these trainee feature specs into `specs/features/trainees` to organise them a little. Are we able to do the same for this spec?

Reply:
Done in e69c4fd